src/SDKs/StorageSync/Management.StorageSync/Generated/Models/TriggerRolloverRequest.cs

```diff
@@ -44,8 +44,8 @@ public TriggerRolloverRequest(string certificateData = default(string))
         /// <summary>
         /// Gets or sets certificate Data
         /// </summary>
-        [JsonProperty(PropertyName = "certificateData")]
-        public string CertificateData { get; set; }
+        [JsonProperty(PropertyName = "serverCertificate")]
+        public string ServerCertificate { get; set; }
```

Comment:
This is a breaking change, please update major version for the nuget package in AssemblyInfo.cs and csproj files